Source/VCGeneration/Counterexample.cs

```diff
@@ -251,6 +240,22 @@ public void PrintModel(TextWriter tw, Counterexample counterexample)
       }
     }
 
+    public void InitializeStates()
+    {
+      if (!Model.ModelHasStatesAlready) {
```

Comment:
I would use `if (Model.ModelHasStatesAlready) { return; }`

Reply:
Resolved in next diff :/ https://github.com/boogie-org/boogie/pull/564/files/e9319c3660598907ec82859218df878a1896fa61..a24711bafc7bf540bf0579ba7e38cce3d6787cb6